synapse/handlers/room_member.py

```diff
@@ -744,6 +747,20 @@ def do_3pid_invite(
             id_server, medium, address
         )
 
+        if not self.spam_checker.user_may_invite(
+            requester.user.to_string(), invitee,
+            third_party_invite={
+                "medium": medium,
+                "address": address,
+            },
+            room_id=room_id,
+            new_room=new_room,
+        ):
+            logger.info("Blocking invite due to spam checker")
+            raise SynapseError(
+                403, "Invites have been disabled on this server",
```

Comment:
surely this is misleading?

Reply:
This is in line with what we say elsewhere for banned invites, but yes. I don't know whether we want to change it now and what to change it to if we did.

Reply:
fair